### PR TITLE
Updating ES to create jks files upon start up if not present in secret dir

### PR DESCRIPTION
**Pull request by ewolinetz**
Should precede https://github.com/openshift/openshift-ansible/pull/9252

### CODE REVIEW

**Comment by richm** [comment]
This will create the following .jks files in /etc/elasticsearch/secret - admin.jks elasticsearch.jks logging-es.jks - I see admin.jks referred to in the elasticsearch.yml: 
```
    openshift.searchguard:
      keystore.path: /etc/elasticsearch/secret/admin.jks
```
but I don't see elasticsearch.jks or logging-es.jks referred to anywhere in the current config or in https://github.com/openshift/openshift-ansible/pull/9252

**Comment by ewolinetz** [comment]
@richm ack, i was creating them to match what ansible created even though we then use different names in the secret. I'll soft link to or mv them with the intended names.

File: elasticsearch/run.sh [hunk 1]
```diff
@@ -87,6 +87,81 @@ sg_role_prometheus:
...
+  if [[ -e $secret_dir/logging-es.key && -e $secret_dir/logging-es.crt ]]; then
+    build_jks_from_pem "logging-es" "logging-es.key" "logging-es.crt" "admin-ca"
+    mv $secret_dir/logging-es.jks $secret_dir/key
+  fi
+
+  if [[ ! -e $secret_dir/truststore ]]; then
+    keytool                                        \
+      -import                                      \
+      -file $secret_dir/admin-ca                   \
+      -keystore $secret_dir/truststore.jks         \
```
Comment (richm) [inline elasticsearch/run.sh]: This file is called just `truststore` in the elasticsearch config:
```
    http:
      truststore_filepath: /etc/elasticsearch/secret/truststore
```

Comment (ewolinetz) [reply elasticsearch/run.sh]: I even had the check above correct... thanks for catching that

**Pull request merged**